website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/step02-create-cluster-fargate.sh

```diff
@@ -0,0 +1,26 @@
+eksctl create cluster -f - << EOF
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+metadata:
+  name: ${CLUSTER_NAME}
+  region: ${AWS_DEFAULT_REGION}
+  version: "1.21"
+  tags:
+    karpenter.sh/discovery: ${CLUSTER_NAME}
+managedNodeGroups:
```

Comment:
Is this Managed Nodegroup necessary when using Fargate?

Reply:
Yeah, I should have removed that. I am having trouble getting just Fargate to work at the moment. I'm debugging right now, so I don't want to merge this yet.

Reply:
I removed the manageNodeGroups from the karpenter example.